_posts/2022-02-14-datafusion-7.0.0.md

```diff
@@ -0,0 +1,166 @@
+    - `corr` [#1561](https://github.com/apache/arrow-datafusion/pull/1561)
+  - Support decimal type [#1394](https://github.com/apache/arrow-datafusion/pull/1394)[#1407](https://github.com/apache/arrow-datafusion/pull/1407)[#1408](https://github.com/apache/arrow-datafusion/pull/1408)[#1431](https://github.com/apache/arrow-datafusion/pull/1431)[#1483](https://github.com/apache/arrow-datafusion/pull/1483)[#1554](https://github.com/apache/arrow-datafusion/pull/1554)[#1640](https://github.com/apache/arrow-datafusion/pull/1640)
+  - Support for evolved schemas [#1622](https://github.com/apache/arrow-datafusion/pull/1622)[#1709](https://github.com/apache/arrow-datafusion/pull/1709)
+  - Support for registering `DataFrame` as table [#1699](https://github.com/apache/arrow-datafusion/pull/1699)
+  - Suppot `substring` function [#1621](https://github.com/apache/arrow-datafusion/pull/1621)
+  - Support `array_agg(distinct ...)` [#1579](https://github.com/apache/arrow-datafusion/pull/1579)
+  - Support `sort` on unprojected columns [#1415](https://github.com/apache/arrow-datafusion/pull/1415)
+- Additional Integration Points
+  - A new public Expression simplification API [#1717](https://github.com/apache/arrow-datafusion/pull/1717)
+- [DataFusion-Contrib](https://github.com/datafusion-contrib)
+  - A new GitHub organization created as a home for both `DataFusion` extensions and as a testing ground for new features.
+    - Extensions
+      - [DataFusion-Python](https://github.com/datafusion-contrib/datafusion-python)
+      - [DataFusion-Java](https://github.com/datafusion-contrib/datafusion-java)
+      - [DataFusion-hdsfs-native](https://github.com/datafusion-contrib/datafusion-hdfs-native)
+      - [DataFusion-ObjectStore-s3](https://github.com/datafusion-contrib/datafusion-objectstore-s3)
+    - New Features
+      - [DataFusion-Streams](https://github.com/datafusion-contrib/datafusion-streams)
+- [Arrow2](https://github.com/jorgecarleitao/arrow2)
+  - An [Arrow2 Branch](https://github.com/apache/arrow-datafusion/tree/arrow2) has been created.  There are ongoing discussions in [DataFusion](https://github.com/apache/arrow-datafusion/issues/1532) and [arrow-rs](https://github.com/apache/arrow-rs/issues/1176) about migrating `DataFusion` to `Arrow2`
+
+For the full list of new features with their relevant PRs, see the
+[enhancements section](https://github.com/apache/arrow-datafusion/blob/7.0.0/datafusion/CHANGELOG.md)
+in the changelog.
+
+# Documentation and Roadmap
+
+The project's documentation is being consolidated into the official site.  You can find more details there on topics such as the SQL status (TO DO LINK) and a user guide.
+
+To provide transparency on DataFusion’s priorities to users and developers a three month roadmap will be published at the beginning of each quarter.  This can be found here (TO DO LINK once site is updated).  
```

Comment:
```suggestion
To provide transparency on DataFusion’s priorities to users and developers a three month roadmap will be published at the beginning of each quarter.  This can be found [here[(https://arrow.apache.org/datafusion/specification/roadmap.html).  
```